crates/bevy_asset/src/path.rs

```diff
@@ -8,7 +10,8 @@ use std::{
 };
 
 /// Represents a path to an asset in the file system.
-#[derive(Debug, Eq, PartialEq, Hash, Clone, Serialize, Deserialize)]
+#[derive(Debug, Eq, PartialEq, Hash, Clone, Serialize, Deserialize, Reflect, FromReflect)]
+#[reflect(Debug, PartialEq, Hash, Serialize, Deserialize, FromReflect)]
 pub struct AssetPath<'a> {
```

Comment:
We also need to register this type in the plugin:

https://github.com/bevyengine/bevy/blob/cb286e5b60ebe8255daaaa5076a3490aa03573e8/crates/bevy_asset/src/lib.rs#L107-L107

Reply:
Sorry about that.

Reply:
No worries! It's my bad for not noticing earlier haha